src/Framework/Framework/Resources/Scripts/utils/logging.ts

```diff
@@ -4,21 +4,37 @@ type LogLevel = "normal" | "verbose";
 
 export const level = getLogLevel();
 
-export function logInfoVerbose(area: string, ...args: any[]) {
+export type DotvvmLoggingArea = (
+    | "debug"
+    | "log"
+    | "postback"
+    | "spa"
+    | "static-command"
+    | "binding-handler"
+    | "resource-loader"
+    | "coercer"
+    | "state-manager"
+    | "validation"
+    | "evaluator"
+    | "events"
+    | "rest-api"
+)
+
+export function logInfoVerbose(area: DotvvmLoggingArea, ...args: any[]) {
     if (compileConstants.debug && level === "verbose") {
         console.log(`%c${area}`, ...args);
     }
 }
 
-export function logInfo(area: string, ...args: any[]) {
+export function logInfo(area: DotvvmLoggingArea, ...args: any[]) {
     console.log(`%c${area}`, ...args);
 }
 
-export function logWarning(area: string, ...args: any[]) {
+export function logWarning(area: DotvvmLoggingArea, ...args: any[]) {
     console.warn(`%c${area}`, ...args);
 }
 
-export function logError(area: string, ...args: any[]) {
+export function logError(area: DotvvmLoggingArea, ...args: any[]) {
     console.error(`%c${area}`, ...args);
```

Comment:
```suggestion
    console.error(area, ...args);
```

Actually, the PR removing colors had to break this. The browser would interpret `args[0]` as a color.